lib/c.c

```diff
@@ -629,12 +629,27 @@ void *malloc(int size)
 
 void *calloc(int n, int size)
 {
-    char *p = malloc(n * size);
+    int total = n * size, offset = 0;
+    char *p = malloc(total);
 
     if (!p)
         return NULL;
-    for (int i = 0; i < n * size; i++)
-        p[i] = 0;
+
+    while ((p + offset) & 3 && offset < total) {
+        p[offset++] = 0;
+    }
```

Comment:
Would it be better to use `memset` instead? However, there is no optimization done in `memset` though.

Reply:
It would, but currently there's no `memset` function currently defined in `lib/c.c`, should I migrate this bulk clearing logic to newly created `memset` function in `lib/c.c`?

Reply:
You can simply add `TODO` comments. Later, we shall implement word-wise `memset`.
See https://github.com/Smattr/memset/blob/master/memset.c